PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/WebAppRunState.java

```diff
@@ -36,6 +36,7 @@
 import com.microsoft.azuretools.utils.WebAppUtils;
 import com.microsoft.intellij.runner.AzureRunProfileState;
 import com.microsoft.intellij.runner.RunProcessHandler;
+import com.microsoft.intellij.runner.container.utils.Constant;
```

Comment:
Wrong place for the constant `LINUX_JAVA_SE_RUNTIME`. Should not be placed under `com.microsoft.intellij.runner.container`.